### PR TITLE
fix: dont load wavelength

### LINKED ISSUE #70: Do not load wavelength from GEANT4 CSV
It is unclear whether the stored wavelength is correct. And in any case, we compute it from tof in our workflows. So we should not load it to begin with.

### PR DESCRIPTION
**Pull request by jokasimr**
Fixes #70

### CODE REVIEW

**Review by jl-wynen — APPROVED** [review]
Please make sure this works with #71 !

**Comment by jokasimr** [comment]
> Please make sure this works with https://github.com/scipp/essdiffraction/pull/71 !

Do you mean make sure to avoid merge conflicts? Or some other aspect?

**Comment by jl-wynen** [comment]
No, to make sure that things don't break at runtime. Basically, don't merge this PR until #71 is merged.

File: tests/dream/io/geant4_test.py
Comment (SimonHeybrock) [inline tests/dream/io/geant4_test.py:115]: This will now cause merge conflicts with #71

Comment (jokasimr) [reply tests/dream/io/geant4_test.py:115]: I'll wait for #71 to be merged before rebasing this

**Pull request merged**
